src/reconcile/rec_visibility.c

```diff
@@ -616,129 +779,8 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, W
     if (upd != NULL)
         r->update_used = true;
 
-    /*
-     * The start timestamp is determined by the commit timestamp when the key is first inserted (or
-     * last updated). The end timestamp is set when a key/value pair becomes invalid, either because
-     * of a remove or a modify/update operation on the same key.
-     *
-     * In the case of a tombstone where the previous update is the ondisk value, we'll allocate an
```

Comment:
The comment
```
     * In the case of a tombstone where the previous update is the ondisk value, we'll allocate an
     * update here to represent the ondisk value. Keep a pointer to the original update (the
     * tombstone) since we do some pointer comparisons below to check whether or not all updates are
     * stable.
     */
```
is no longer present in the code. Is it no longer applicable?

Reply:
Yes, I think this is no longer applicable.